tests/app/shared/issue-tables/issue-tables.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { IssueTablesComponent } from '../../../../src/app/shared/issue-tables/issue-tables.component';
+
+describe('IssueTablesComponent', () => {
+    let issueTablesComponent: IssueTablesComponent;
+
+    describe('fitTitleText()', () => {
+        beforeEach(() => {
+        issueTablesComponent = new IssueTablesComponent(null, null, null, null, null, null, null);
```

Comment:
Slight indentation error here not caught by the linter! o.o 

Will be good to fix!

Reply:
Thanks for pointing it out!